neon_core/__init__.py

```diff
@@ -55,25 +62,23 @@ def setup_holmes_config():
             "neon_enclosure": "neon_core"
         }
     }
+
     cfg = {}
-    if exists(HOLMES_CONFIG):
-        try:
-            with open(HOLMES_CONFIG) as f:
-                cfg = json.load(f)
-        except:
-            pass
-    cfg = merge_dict(cfg, _NEON_HOLMES_CONFIG)
-    with open(HOLMES_CONFIG, "w") as f:
-        json.dump(cfg, f, indent=4, ensure_ascii=True)
+    try:
+        with open(OVOS_CONFIG) as f:
+            cfg = json.load(f)
+    except Exception as e:
+        LOG.error(e)
```

Comment:
isnt this too spammy if file doesnt exist? feels like should only be logged in exceptions other than FileNotFound

Reply:
Makes sense; I just don't like catch-all errors that don't log or anything. I'll handle FNF without logging